docs/src/main/sphinx/release/release-421.md

```diff
@@ -0,0 +1,66 @@
+# Release 421 (5 Jul 2023)
+
+## General
+
+* Add support for check constraints in an `UPDATE` statement. ({issue}`17195`)
+* Improve performance for queries involving a `year` function within an `IN`
+  predicate. ({issue}`18092`)
+* Fix query failure when cancelling a query with a window function. ({issue}`18061`)
+* Fix query failure for queries involving the `concat_ws` function on arrays
+  with more than 254 values. ({issue}`17816`)
+* Fix loss of timestamp precision when coercing a
```

Comment:
This doesn't seem to match what the PR is about. What's the observed behavior before that PR? A failure? Correctness issues? cc @Praveen2112